### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by marcusesa**
# Setup project

Adding silex, monolog and initial build artefacts.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+vendor
+/log/*.log
+.idea/
+/build
+!/build/.gitkeep
```
Comment (valentim) [inline .gitignore:5]: I think that it is interesting to add ".vagrant", "_.swp" and "_.swo"

**Comment by marcelioleal** [comment]
great

**Comment by raulbcs** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (marcusesa) [inline .gitignore:5]: thanks @valentim

**Pull request merged**